vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -15,6 +15,17 @@
 					      	<div style="height: 5px; width: 5px; border-radius:50%; background-color: var(--success)"></div>
 					      	<em style="color: var(--dark-grey); font-size: 12px; margin-left: 10px">logged in as <span th:text="${#authentication.name}"></span></em>
 					    </div>
+		        	</li>
+		        	<a  href="/vspace/exhibit/aboutPage">
```

Comment:
this link won't work e.g. on staging since the app isn't called 'vspace' there. This needs to use thymeleaf's mechanism to create links.

Reply:
Changed